packages/ui/src/components/numpad/numpad.native.tsx

```diff
@@ -0,0 +1,159 @@
+import { ReactNode } from 'react';
```

Comment:
This file is quite a monster. Maybe it could be split up a bit? Separating logic from presentation

Reply:
Agreed. Making a note to extract the logic in the upcoming iteration, along with other tweaks.